src/core/render/compiler.js

```diff
@@ -222,7 +222,7 @@ export class Compiler {
 
       if (/<!-- {docsify-ignore-all} -->/g.test(str)) {
         str = str.replace('<!-- {docsify-ignore-all} -->', '');
-        nextToc.title = str;
+        nextToc.title = removeAtag(str);
```

Comment:
There are still a few things that need to be changed.

https://github.com/docsifyjs/docsify/blob/c720928b9e2b8e41ab63ea324cfd061000899a57/src/core/render/compiler.js#L217-L221

https://github.com/docsifyjs/docsify/blob/c720928b9e2b8e41ab63ea324cfd061000899a57/src/core/render/compiler.js#L229-L233

https://github.com/docsifyjs/docsify/blob/c720928b9e2b8e41ab63ea324cfd061000899a57/src/core/render/compiler/headline.js#L4-L31

Reply:
the PR is before the patch(pending a long time), I will fix it.

Reply:
`docsify/src/core/render/compiler/headline.js` It still needs to be modified.

Reply:
k.